js/index.js

```diff
@@ -42,8 +42,40 @@ window.addEventListener('load', event => {
   // Bind the click event of each element to a function
   document.querySelectorAll('.card').forEach(card => {
     card.addEventListener('click', () => {
-      // TODO: write some code here
-      console.log(`Card clicked: ${card}`);
+      let cardName = card.getAttribute("data-card-name");
+      console.log(`card picked is ${cardName}`);
+      card.querySelectorAll('.card div').forEach(div =>{
+        div.classList.toggle("front")
+        div.classList.toggle("back")
+      })
+      console.log(`bla`,memoryGame.pickedCards)
+      memoryGame.pickedCards.push(cardName)
+      console.log(`nlanla`,memoryGame.pickedCards)
+      let card1 = memoryGame.pickedCards[0];
+      let card2 = memoryGame.pickedCards[1];
+      //console.log(memoryGame.checkIfPair(memoryGame.pickedCards[0],memoryGame.pickedCards[1]))
+      if(memoryGame.pickedCards.length === 2){
```

Comment:
After line 57 is a good moment to check for both cards' name, you can add two variables for that and then use them in your next conditional statement.

Also, since we have defined `checkIfPair()` to return either `true` or `false` then there's no need  for `=== true`.
```
const card1Name = card1.getAttribute("data-card-name");
const card2Name = card2.getAttribute("data-card-name");

if(memoryGame.checkIfPair(card1Name, card2Name)){...}